IDE/Renesas/e2studio/RX72N/Readme.md

```diff
@@ -0,0 +1,303 @@
+You can download and excute wolfBoot by e2Studio debugger. Use a USB connection between PC and the
+board for the debugger and flash programmer.
+
+
+### 3-1 Key generation
+
+```
+$ cd <wolfBoot>
+$ export PATH:$PATH:<wolfBoot>/tools/keytools
+$ keygen --ecc256 -g ./pri-ecc256.der
+```
+
+It generates a pair of private and public key with -g option. The private key is stored 
+in the specified file. The public key is stored in a key store as a C source code 
+in "src/keystore.c" soo that it can be compiled and linked with wolfBoot.
+If you have an existing key pair, you can use -i option to import the pablic
+key to the store.
+
+You can specify various signature algorithms such as 
+
+```
+--ed25519 --ed448 --ecc256 --ecc384 --ecc521 --rsa2048 --rsa3072
+```
+
+### 3-2 Compile wolfBoot
+
+Open project under IDE/Renesas/e2studio/RX72N/wolfBoot with e2Studio, and build the project.
+
+Project properties are preset for the demo.
+
+```
+Smart Configurator
+Flash Driver: r_flash_rx
+
+Include Paths
+../../include : <wolfBoot>/IDE/Renesas/e2studio/RX72N/include
+../../../../../../include : <wolfBoot>/include
+../../../../../../lib/wolfssl/ : <wolfBoot>/lib/wolfssl
+
+Pre-Include
+../../include/user_settings.h : <wolfBoot>/IDE/Renesas/e2studio/RX72N/include/user_settigs.h
+../../include/target.h : <wolfBoot>/IDE/Renesas/e2studio/RX72N/include/target.h
+
+Pre-defined Pre-processor Macro
+__WOLFBOOT
+WOLFBOOT_PARTIION_INFO
+PRINTF_ENABLED
+
+```
+
+WOLFBOOT_PARTION_INFO, PRINTF_ENABLED are for debug information about partitions.
+Eliminate them for operational use.
+
+
+### 3-4 Compile the sample application
+
+Open project under IDE/Renesas/e2studio/RX72N/app_RenesasRx01 with e2Studio, and build the project.
+
+
+Project properties are preset for the demo.
+
+```
+Smart Configurator
+Flash Driver: r_flash_rx
+
+Include Paths
+Include Paths
+../../include : <wolfBoot>/IDE/Renesas/e2studio/RX72N/include
+../../../../../../include : <wolfBoot>/include
+
+Pre-Include
+../../include/user_settings.h : <wolfBoot>/IDE/Renesas/e2studio/RX72N/include/user_settigs.h
+../../include/target.h : <wolfBoot>/IDE/Renesas/e2studio/RX72N/include/target.h
+
+Code Origin and entry point (PResetPRG) is "0xffc10200" (See Section Viewer of Linker Section).
+```
+
+app_RenesasRx01.x in ELF is gnerated under HardwareDebug. You can derive bair binary file 
+(app_RenesasRx01.bin) by rx-elf-objcopy.exe command as follows. -R are for eliminate unnecessary
+secrions.
+
+```
+$ rx-elf-objcopy.exe -O binary\
+  -R '$ADDR_C_FE7F5D00' -R '$ADDR_C_FE7F5D10' -R '$ADDR_C_FE7F5D20' -R '$ADDR_C_FE7F5D30'\
+  -R '$ADDR_C_FE7F5D40' -R '$ADDR_C_FE7F5D48' -R '$ADDR_C_FE7F5D50' -R '$ADDR_C_FE7F5D64'\
+  -R '$ADDR_C_FE7F5D70' -R EXCEPTVECT -R RESETVECT app_RenesasRx01.x app_RenesasRx01.bin
+```
+
+### 3-5 Generate Signature for app V1
+
+"sign" command under tools/keytools benerates a signature for the binary with a specified version.
```

Comment:
_The_ "sign" command. (or perhaps better formatted as code: "The `sign` command"

Reply:
"generates"